test/extensions/filters/network/redis_proxy/conn_pool_impl_test.cc

```diff
@@ -52,9 +53,12 @@ class RedisClientImplTest : public testing::Test, public Common::Redis::DecoderF
   ~RedisClientImplTest() {
     client_.reset();
 
-    // Make sure all gauges are 0.
+    // Make sure all gauges are 0, except those in circuit_breakers which default
+    // to the resource max
     for (const Stats::GaugeSharedPtr& gauge : host_->cluster_.stats_store_.gauges()) {
-      EXPECT_EQ(0U, gauge->value());
+      if (!absl::StrContains(gauge->name(), "circuit_breakers")) {
```

Comment:
I think this whitelisting is fine, thought it might make sense at one point to add a `initialValue()` function to `Gauge` so that these checks can just be `EXPECT_EQ(gauge->initialValue(), gauge->value())`, but that would arguably fall out of the scope of this PR

Reply:
I will defer to your (or anyone else's) judgment on whether or not to make that change. It makes sense to me.